examples/run_largecyc_CPMG.py

```diff
@@ -0,0 +1,224 @@
+        + 1e3 * config_dict["echo_acq_ms"]
+        + prog_p180_us
+    )
+)
+# }}}
+# {{{check total points
+total_pts = nPoints * nPhaseSteps * config_dict["nEchoes"]
+assert total_pts < 2**14, (
+    "You are trying to acquire %d points (too many points) -- either change SW or acq time so nPoints x nPhaseSteps is less than 16384"
+    % total_pts
+)
+# }}}
+# {{{ acquire CPMG
+# NOTE: Number of segments is nEchoes * nPhaseSteps
+data = generic(
+    ppg_list=[
+        ("phase_reset", 1),
+        ("delay_TTL", config_dict["deblank_us"]),
+        ("pulse_TTL", prog_p90_us, "ph_cyc", ph1_cyc),
+        (
+            "delay",
+            config_dict["tau_us"]
+            - 2 * prog_p90_us / pi
+            - marker_us
+            - config_dict["deblank_us"],
+        ),
+        ("delay", marker_us),  # placeholder for marker
+        ("delay_TTL", config_dict["deblank_us"]),
+        ("pulse_TTL", prog_p180_us, "ph_cyc", ph3_cyc),
+        ("delay", config_dict["deadtime_us"]),
+        ("acquire", config_dict["echo_acq_ms"]),
+        (
+            "delay",
+            config_dict["deadtime_us"]
+            - 2 * marker_us
+            - config_dict["deblank_us"],
+        ),
+        ("delay", marker_us),  # placeholder for jumpto
+        # NOTE: here the tau_us is defined as
+        # the evolution time from the start of
+        # excitation (*during the pulse*) through
+        # to the start of the 180 pulse
+        ("marker", "echo_label", config_dict["nEchoes"] - 1),
+        ("delay_TTL", config_dict["deblank_us"]),
+        ("pulse_TTL", prog_p180_us, "ph_cyc", ph3_cyc),
+        ("delay", config_dict["deadtime_us"]),
+        ("acquire", config_dict["echo_acq_ms"]),
+        (
+            "delay",
+            config_dict["deadtime_us"]
+            - 2 * marker_us
+            - config_dict["deblank_us"],
+        ),
+        ("jumpto", "echo_label"),
+        # In the line above I assume this takes
+        # marker_us to execute The way to be sure
+        # of this would be to capture on a scope
+        # and measure from one 180 to the next (or
+        # actually several, since this error would
+        # be cumulative
+        ("delay", config_dict["repetition_us"]),
+    ],
+    nScans=config_dict["nScans"],
+    indirect_idx=0,
+    indirect_len=1,
+    adcOffset=config_dict["adc_offset"],
+    carrierFreq_MHz=config_dict["carrierFreq_MHz"],
+    nPoints=nPoints,
+    time_per_segment_ms=config_dict["echo_acq_ms"],
+    SW_kHz=config_dict["SW_kHz"],
+    ret_data=None,
+    manual_echoes=1,
+)
+# }}}
+# {{{ chunk and save data
+data.chunk(
+    "t",
+    ["ph2", "ph_diff", "nEcho", "t2"],
+    [len(ph2), len(ph_diff), config_dict["nEchoes"], -1],
+)
+data.setaxis("nEcho", r_[0 : config_dict["nEchoes"]]).setaxis(
+    "ph2", ph2 / 4
+).setaxis("ph_diff", ph_diff / 4)
+data.set_prop("postproc_type", "spincore_diffph_SE_v2")
+data.set_prop("coherence_pathway", {"ph_overall": -1, "ph1": +1})
```

Comment:
new coherence pathway needed

Reply:
yes, let's just go with:
```
{'ph1': +1,
'ph2": -2,
'ph_overall':-1,
}
```
note that this will only pull the odd echoes, but we have  to pick something for now.

(we can make this more complicated by using "eval", but that's beyond this PR)